ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -80,6 +80,7 @@ type RayClusterReconciler struct {
 // +kubebuilder:rbac:groups="rbac.authorization.k8s.io",resources=roles,verbs=get;list;watch;create;delete;update
 // +kubebuilder:rbac:groups="rbac.authorization.k8s.io",resources=rolebindings,verbs=get;list;watch;create;delete
 
+// [WARNING]: There MUST be a newline after kubebuilder markers.
```

Comment:
em. is not line 82 a new line? I may forgot the tricks here.

Reply:
The new line is added by #577. Originally, it does not have a newline.

Reply:
I mean the new line was added in https://github.com/ray-project/kuberay/pull/577 and the bug should be fixed? This PR is to add some comment to reminder developer?  But the title is the bug fix?

Reply:
Got it. The title is confusing. I opened this PR prior to #577, but #577 was merged prior to this PR. Then, I rebased this branch with upstream. I will update the title. Thank you for pointing this out!